build.assets/Dockerfile

```diff
@@ -271,5 +271,10 @@ RUN cd /usr/local/lib && \
     ldconfig
 COPY pkgconfig/buildbox/ /
 
+# Install latest git. git 2.18+ is required for GitHub Actions.
+RUN apt-get -y update && apt-get -y install software-properties-common && \
```

Comment:
`git` is installed above on line 103 from the repositories - should it be removed from there?